### DISCUSSION TITLE
Is there an "auto comment on enter" feature?

### DISCUSSION THREAD

**Discussion by Multirious** [helix-editor/helix#7952] [Q&A] [answered]
It'd be very convenient for me or other people who writes a lot of code documents.
I usually do the Rust's `//!` thing which is not `C-c` able. A lot of times I did the multi cursor edit to add the `//!` which gets tedious.

**Comment by woojiq** [comment, marked as answer]
Duplicate of https://github.com/helix-editor/helix/issues/1730

**Reply by Multirious** [reply]
> [Oops, I somehow missed that, even with issue search for comment/comments. Sorry. I blame GitHub search.](https://github.com/helix-editor/helix/issues/7284#issuecomment-1581951194)

Oh, I'm sorry too. Somehow can't search for the issue too.
